.github/scripts/runner_determinator.py

```diff
@@ -198,6 +198,13 @@ def parse_args() -> Any:
         default="",
         help="comma separated list of experiments to check, if omitted all experiments marked with default=True are checked",
     )
+    parser.add_argument(
+        "--opt-out-experiments",
+        type=_str_comma_separated_to_set,
+        required=False,
+        default="",
+        help="comma separated list of experiments to opt-out of. If unset, no opt-outs will occur.",
```

Comment:
Nit: Could you please also clarify in the help text the resolution policy when an experiment is both opted out and marked as "eligible-experiments" (we exclude it)